columnar_batch/columnar_batch.h

```diff
@@ -1,6 +1,9 @@
 #include "../fusion_power_video.h"
 
-namespace fpvc {
+#ifndef COLUMNAR_BATCH_H_
```

Comment:
Does this also work with the include guards starting at the very top above the include?